package.json

```diff
@@ -67,7 +67,7 @@
   "dependencies": {
     "aws-cdk-github-oidc": "^2.4.1",
     "aws-cdk-lib": "2.139.0",
-    "cloudstructs": "^0.9.5",
+    "cloudstructs": "^0.9.6",
```

Comment:
Consider upgrading the `cloudstructs` dependency to the latest version `0.9.6` to take advantage of the latest bug fixes and improvements. [medium]